build.R

```diff
@@ -211,6 +243,16 @@ plots <- list(
       displayName = "t-statistics",
       plotType = c("multiFeature", "multiTest"),
       packages = c("gplots", "viridis")
+    ),
+    single_feature_plotly = list(
+      displayName = "Expression by cell type interactive with plotly",
+      plotType = "singleFeature",
+      packages = c("ggplot2", "viridis", "plotly")
+    ),
+    heatmap.plotly = list(
+      displayName = "Expression Heatmap interactive with plotly",
+      plotType = "multiFeature",
+      packages = c("viridis", "plotly")
     )
   )
 )
```

Comment:
Could you please also add some calls to `plotStudy()` with these new plotting functions? You should be able to copy-paste-modify the existing singleFeature and multiFeature static plot examples below. Also, since it's just going to spew json, it would probably be best to assign the output to a variable, e.g. `jsonBoxplot` and `jsonHeatmap`